config/defaults.js

```diff
@@ -77,6 +77,10 @@ const defaultConfig = {
 				"music_offtopic",
 			],
 		},
+		"video-toggle": {
+			enabled: false,
```

Comment:
nit: we could  add a migration so that people who had the old plugin enabled have this one enabled in the new version.